.travis.yml

```diff
@@ -49,6 +49,8 @@ script:
 after_success:
   - cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./coverage
 
+before_deploy: npm run pre-release
```

Comment:
I'd replace that with `npms` `prepublishOnly` of possible. (https://docs.npmjs.com/misc/scripts)